sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -72,7 +71,7 @@ def test_sanity_check_pls_regression():
     Y = d.target
 
     pls = PLSRegression(n_components=X.shape[1])
-    pls.fit(X, Y)
+    pls.fit_transform(X, Y)
```

Comment:
The statement regarding transform is true for `X` but not `y`:

```python
from sklearn.datasets import load_linnerud
from numpy.testing import assert_allclose

X, y = load_linnerud(return_X_y=True)
pls = PLSRegression(n_components=X.shape[1])
pls.fit(X, y)

X_trans, y_trans = pls.fit_transform(X, y)
assert_allclose(X_trans, pls.x_scores_)
```

So we can still test for `X_trans`.